src/LabLog/Views/Teacher/Index.cshtml

```diff
@@ -1,9 +1,13 @@
-﻿@{
-    ViewData["Title"] = "List of Rooms";
-}
+﻿@model UserModel
 
-
-<p>This page is the admin landing page.</p>
-<p>It will contain a list of rooms, as well as
-    the options to create a new room. Room editing and deleting will be done from
-    within each Room's view page.</p>
\ No newline at end of file
+  <div class="wrapper">
+    <form class="form-signin" action="/Teacher" method="POST">
+      <h2 class="form-signin-heading">Please login</h2>
+      <input type="text" class="form-control" name="username" placeholder="E-Number" required="True" autofocus="" />
+      <input type="password" class="form-control" name="password" placeholder="Password" required="True"/>      
+      <label class="checkbox">
+        <input type="checkbox" value="remember-me" id="rememberMe" name="rememberMe"> Remember me
+      </label>
+      <button class="btn btn-lg btn-primary btn-block" type="submit">Login</button>   

```

Comment:
Definitely use the tag helpers with the button, it'll link it to a particular controller and action and allow for easy refactoring